### ISSUE TITLE
Image Uploads

### ISSUE THREAD

**Issue by TimCliff** [steemit/condenser#396]
I'm not sure the best way to implement this. Most likely the images would need to be stored somewhere off the actual Steem blockchain. Opening for discussion.

**Comment by sneak** [comment]
In progress elsewhere.  Stay tuned.

**Comment by sneak** [comment]
(note that I am closing this not because we don't like the feedback, but that the steemit.com repository is specifically for issues related to the code inside of it - it's not a discussion forum.)

**Comment by jcalfee** [comment]
Add a simple github-like image upload in the markdown editor.  An upload button is probably the most likely scenario so that would be the first method.  While uploading show a loading indicator (progress bar perhaps) outside of the editor and when done insert the markdown text with the image url in the editor.

**Comment by jcalfee** [comment]
Did I mention this would be like GitHub?  lol .. Well here it is, feel free to change it but this is the basic idea..

This file I used the selecting them link to choose the file on my hard drive:

Submit a Story:

![image](https://cloud.githubusercontent.com/assets/204121/21109895/a9cfaf00-c060-11e6-8fb6-6c1750c4325f.png)

This is a Reply to a post or comment.  I did a screen capture and pasted from clipboard but I can also open images and copy / paste them into the editor..  I don't think we want that text bar as shown above on the reply box, it is bulky.  Having that description on "Submit a Story" will help users find this feature.

![image](https://cloud.githubusercontent.com/assets/204121/21109944/e868137e-c060-11e6-9184-714dc8f4501c.png)
